COMET.Web.Common/App.razor

```diff
@@ -43,6 +44,5 @@
             </LayoutView>
         </NotFound>
     </Router>
+    <AntContainer />
```

Comment:
Is it required to do it? I would rather not include any dependency to AntDesign to the common library...

Reply:
AntDesign documentation recommends to add the <AntContainer/> tag inside App.razor: https://antblazor.com/en-US/docs/introduce

Reply:
Closing this as AntDesign is not in common lib anymore